src/devices/Tca9548a/Tca9548A.cs

```diff
@@ -0,0 +1,156 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Device.I2c;
+using System.Device.Model;
+using System.IO;
+using Iot.Device.Board;
+
+namespace Iot.Device.Tca9548a
+{
+    /// <summary>
+    /// Tca9548A - 8-Channel I2C Switch/Multiplexer
+    /// </summary>
+    [Interface("BNO055 - 8-Channel I2C Switch/Multiplexer")]
```

Comment:
please adjust the interface name here ;-)
```suggestion
    [Interface("Tca9548A - 8-Channel I2C Switch/Multiplexer")]
```